src/main/scala/mesosphere/marathon/core/groupmanager/package.scala

```diff
@@ -0,0 +1,6 @@
+package mesosphere.marathon.core
+
+/**
+  * This package allows to manage all group related actions.
+  */
+package object groupmanager
```

Comment:
The service `GroupManager` now is in package `groupmanager`.
We should find a better name for the package. 
Could be as simple as `group`